NodeBase/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM selenium/base:2.52.0
+FROM selenium/base:latest
```

Comment:
Okay, fair enough.. I'll try to get to this and address your comments later today.